.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
+
+on: pull_request
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228449512-bc7cb77d-53e8-4f6f-b97b-e38abe4d2889.png)

**[OPTIONAL]**

- [ ] You have a branch name that is descriptive 👍 but you can make it more descriptive such as `feature/add-data-model` to make your branch name more professional and descriptive 😊

Reply:
Thank you for this suggestion 👍 . Moving forward, I'll have branch names which are more descriptive 💯 . And thanks for the review and feeback 🙏